packages/botonic-react/src/components/carousel.jsx

```diff
@@ -16,33 +30,17 @@ export const Carousel = props => {
   let content = props.children
   if (isBrowser()) {
     content = (
-      <div
-        style={{
-          paddingTop: '10px',
-          display: 'flex',
-          flexDirection: 'row',
-          overflowX: 'auto',
-          maxWidth: '100%',
-        }}
-      >
-        <div
-          style={{
-            display: 'flex',
-            flexDirection: 'row',
-            alignItems: 'start'
-          }}
-        >
-          {props.children}
-        </div>
-      </div>
+      <Box>
+        <Item>{props.children}</Item>
+      </Box>
     )
   }
   return (
     <Message
       style={{ maxWidth: '90%', padding: 0 }}
       blob={false}
       json={serialize(props)}
-      type='carousel'
+      type="carousel"
```

Comment:
did you get an error with single quotes? @Khaeshah shared some prettier config to use " on jsx